src/transformers/modeling_utils.py

```diff
@@ -2795,11 +2828,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 offload_folder=offload_folder,
                 offload_state_dict=offload_state_dict,
                 dtype=torch_dtype,
-                load_in_8bit=load_in_8bit,
+                load_in_kbit=(load_in_8bit or load_in_4bit),
                 keep_in_fp32_modules=keep_in_fp32_modules,
             )
 
+        model.is_loaded_in_4bit = load_in_4bit
         model.is_loaded_in_8bit = load_in_8bit
+        model.is_loaded_in_kbit = load_in_8bit or load_in_4bit
```

Comment:
`is_loaded_in_kbit` is not a good name. All models are lodaded in a certain kbit precision for k in {4, 8, 16, 32}. `is_quantized` would be more adapted.